automation-tests/test/specs/orgCreationAndAuthentication.e2e.ts

```diff
@@ -0,0 +1,261 @@
+    await utilities.pause(1);
+
+    // In the drop down menu that appears, select "vscodeOrg - user_name".
+    await utilities.selectQuickPickItem(prompt, `${EnvironmentSettings.getInstance().devHubAliasName} - ${EnvironmentSettings.getInstance().devHubUserName}`);
+
+    // Need to pause here for the "set a default org" command to finish.
+    await utilities.pause(5);
+
+    // Look for the notification that appears which says, "SFDX: Set a Default Org successfully ran".
+    const successNotificationWasFound = await utilities.notificationIsPresent(workbench, 'SFDX: Set a Default Org successfully ran');
+    expect(successNotificationWasFound).toBe(true);
+
+    const expectedOutputWasFound = await utilities.attemptToFindOutputPanelText('Salesforce CLI', `defaultusername ${EnvironmentSettings.getInstance().devHubAliasName} true`, 5);
+    expect(expectedOutputWasFound).not.toBeUndefined();
+
+    // Look for "vscodeOrg" in the status bar.
+    const vscodeOrgItem = await statusBar.getItem(`plug  ${EnvironmentSettings.getInstance().devHubAliasName}, Change Default Org`);
+    expect(vscodeOrgItem).not.toBeUndefined();
+  });
+
+  step('Run SFDX: Create a Default Scratch Org', async () => {
+    const workbench = await (await browser.getWorkbench()).wait();
+    await utilities.executeQuickPick(workbench, 'SFDX: Create a Default Scratch Org...');
+
+    // Select a project scratch definition file (config/project-scratch-def.json)
+    // Press Enter/Return to use the default (config/project-scratch-def.json)
+    await prompt.confirm();
+
+    // Enter an org alias - yyyy-mm-dd-username-ticks
+    const currentDate = new Date();
+    const ticks = currentDate.getTime();
+    const day = ("0" + currentDate.getDate()).slice(-2);
+    const month = ("0" + (currentDate.getMonth() + 1)).slice(-2);
+    const year = currentDate.getFullYear();
+    const userName = utilities.currentUserName();
+    scratchOrgAliasName = `TempScratchOrg_${year}_${month}_${day}_${userName}_${ticks}_OrgAuth`;
+
+    await prompt.setText(scratchOrgAliasName);
+    await utilities.pause(1);
+
+    // Press Enter/Return.
+    await prompt.confirm();
+
+    // Enter the number of days.
+    await prompt.setText('1');
+    await utilities.pause(1);
+
+    // Press Enter/Return.
+    await prompt.confirm();
+
+    await utilities.waitForNotificationToGoAway(workbench, 'Running SFDX: Create a Default Scratch Org...', 5 * 60);
+
+    const successNotificationWasFound = await utilities.notificationIsPresent(workbench, 'SFDX: Create a Default Scratch Org... successfully ran');
+    if (successNotificationWasFound != true) {
+      const failureNotificationWasFound = await utilities.notificationIsPresent(workbench, 'SFDX: Create a Default Scratch Org... failed to run');
+      if (failureNotificationWasFound == true) {
+        if (await utilities.attemptToFindOutputPanelText('Salesforce CLI', 'organization has reached its daily scratch org signup limit', 5)) {
```

Comment:
Nice job handling these known issues that can happen during scratch org creation.